test/aleph/http_test.clj

```diff
@@ -180,25 +180,25 @@
            (.shutdown *pool*))))))
 
 (defmacro with-handler [handler & body]
-  `(with-server (http/start-server ~handler {:port port})
+  `(with-server (http/start-server ~handler {:port port :shutdown-timeout 0})
```

Comment:
Wouldn't it be simpler to add this to the `default-options` map at the top?

Reply:
`default-options` at the top is used for HTTP clients, not servers.

Reply:
We should probably make one for servers, then. Still, if you're sick of this, it doesn't have to be in this PR 😄

Reply:
I'll re-prepare the commits this morning and then will add this.

Reply:
After a second read I think I will not tackle this refactor for this PR